src/app/service/photos/photos.service.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class PhotosService {
+
+  constructor(private http: HttpClient) { }
+
+  getPhotos(): Promise<Photo[]> {
+    return this.http.get(`https://${environment.facebookphotosUrl}`, {
```

Comment:
Like the above `environment` is also an external declaration. Please fix this by adding `import { environment } from '../../../environments/environment';` to the top of the file.